src/Microsoft.DotNet.Compiler.Common/AssemblyInfoFileGenerator.cs

```diff
@@ -79,6 +79,11 @@ public static string GenerateFSharp(AssemblyInfoOptions metadata)
                 attributes[typeof(TargetFrameworkAttribute)] = EscapeCharacters(metadata.TargetFramework);
             };
 
+            if (metadata.IsServiceable)
+            {
+                attributes[typeof(AssemblyMetadataAttribute)] = "Serviceable\", \"True";
```

Comment:
@eerhardt because the rest of the attributes are handled this way, so I'm just following the current convention

@brthor because AssemblyMetadataAttribute is a Key-Value pair whereas the code that consumes this assumes only one string value